Misc/NEWS.d/next/Core and Builtins/2017-08-28-11-51-29.bpo-31291.t8QggK.rst

```diff
@@ -0,0 +1,3 @@
+Fix an assertion failure in `zipimport.zipimporter.get_data` on Windows,
+when the return value of pathname.replace('/','\\') isn't a string. Patch by
```

Comment:
Format `pathname.replace('/','\\')` as a literal code (surround it with double backquotes). Otherwise `\\` will be interpreted in reST.